### ISSUE TITLE
dimension is null (Cannot read property 'attrib' of null) 

### ISSUE THREAD

**Issue by atomantic** [optilude/xlsx-template#30]
When trying to add a table with multiple rows I get this error: 
TypeError: Cannot read property 'attrib' of null

traced down to this:
```
dimension = sheet.root.find("dimension")
```
dimension turns out to be null
```
TypeError: Cannot read property 'attrib' of null
    at Workbook.module.exports.Workbook.substitute (.../node_modules/xlsx-template/lib/index.js:248:59)
```

**Comment by atomantic** [comment]
I commented out this block: https://github.com/optilude/xlsx-template/blob/master/lib/index.js#L243-L252

and it works fine--so I imagine you could wrap that whole thing in a `if(dimension){}` :)

**Comment by kant2002** [comment]
Fixed in 0.1.1
